### DISCUSSION TITLE
Guest user

### DISCUSSION THREAD

**Discussion by bruchar1** [navidrome/navidrome#2313] [Ideas]
I think it could be great to have to possibility to log in without a user. This could be displayed as a "Log as guest" link on the login page. Each guest user would be isolated from others, and modifications (votes, playlists, etc.) would not be persistent.

**Comment by deluan** [comment]
Sorry, but this is not currently supported. We may add it in the future, but don't hold your breath :)

**Comment by enekonieto** [comment]
Our use case: we are a 20 people group sharing an intranet. Most of them are not tech savvy and we trust each other. 

Solution: autologin to shared `dj` user adding `DevAutoLoginUsername = "dj"` to `/var/lib/navidrome/navidrome.toml`
